### ISSUE TITLE
Add install instructions to README

### ISSUE THREAD

**Issue by franticspider** [laurikari/tre#30]
This is a great library, but I'd like to know how to build from source rather than install from the Ubuntu repository. Please add instructions for how to build to the README (or to an INSTALL file)

thanks,

simon

**Comment by shawnl** [comment]
just do (from https://www.debian.org/doc/manuals/maint-guide/build.en.html )`
apt-get install build-essential
apt-get build-dep tre
apt-get source tre
cd tre-*
dpkg-buildpackage
`
Otherwise building is documented in README.darcs

**Comment by franticspider** [comment]
Thanks for that, but I need to build the library on a system where I do not have root privileges. I was hoping to use automake or cmake or somesuch to build the library. The README doesn't tell me how to do this.

**Comment by cmarqu** [comment]
To compile from source into a user-local directory, do something like
```
./utils/autogen.sh
./configure --prefix=$HOME/tools/
make
make install
```

**Comment by dag-erling** [comment]
Fixed in 1083bcb088bc.
